diff --git a/src/Select.svelte b/src/Select.svelte
index 4650498459afff848ba0578dab0001508875c62c..410a6ce3fde99f2de3df19f0a96970e3690199db 100644
--- a/src/Select.svelte
+++ b/src/Select.svelte
@@ -333,7 +333,7 @@
   export let noLabel = false;
   export let label: string | undefined = undefined;
   export let value: any = '';
-  export let key: (item: any) => string = (item) => item;
+  export let key: (item: any) => string = (item) => item?.toString?.();
   export let dirty = false;
 
   // Some trickery to detect uninitialized values but also have the right types.
