diff --git a/dist/Select.svelte b/dist/Select.svelte
index f2e1bbbb6e57dc1d2f87971bd7a5c72361421ea0..efe0ebc47cd0c0b1f7e52ae2b539ecf0cc47b180 100644
--- a/dist/Select.svelte
+++ b/dist/Select.svelte
@@ -251,7 +251,7 @@ export let variant = 'standard';
 export let noLabel = false;
 export let label = undefined;
 export let value = '';
-export let key = (item) => item;
+export let key = (item) => item?.toString?.();
 export let dirty = false;
 // Some trickery to detect uninitialized values but also have the right types.
 export let invalid = uninitializedValue;
